### PR TITLE
W2-D2-thachnguyen-Activity Fragment

### PR DESCRIPTION
**Pull request by at-thachnguyen2**
Task: #14

### CODE REVIEW

File: app/src/at-thachnguyen/java/asiantech/internship/summer/thachnguyen/debug/activity_and_fragment/LogInFragment.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            SignUpFragment signUpFragment = new SignUpFragment();
+            FragmentTransaction transaction = getActivity().getSupportFragmentManager().beginTransaction();
+            transaction.setCustomAnimations(R.anim.slide_right_in, R.anim.slide_left_out,
+                    R.anim.slide_left_in, R.anim.slide_right_out);
+            transaction.replace(R.id.fragmentContainer, signUpFragment);
+            transaction.addToBackStack(null);
+            transaction.commit();
+        });
+
+        tvLogIn.setOnClickListener((View view) -> {
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/java/asiantech/internship/summer/thachnguyen/debug/activity_and_fragment/LogInFragment.java]: @at-thachnguyen2 : please edit code in this function for terse

File: app/src/at-thachnguyen/java/asiantech/internship/summer/thachnguyen/debug/activity_and_fragment/SignUpFragment.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
+        View contentView = inflater.inflate(R.layout.fragment_sign_up, container, false);
+        EditText edtEmail = contentView.findViewById(R.id.edtEmail);
+        EditText edtPassword = contentView.findViewById(R.id.edtPassword);
+        EditText edtConfirmPassword = contentView.findViewById(R.id.edtConfirmPassword);
+        TextView tvSignUp = contentView.findViewById(R.id.tvSignUp);
+        Toolbar toolbar = Objects.requireNonNull(getActivity()).findViewById(R.id.toolbar);
+        toolbar.setTitle("Sign up");
+
+        tvSignUp.setOnClickListener(view -> {
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/java/asiantech/internship/summer/thachnguyen/debug/activity_and_fragment/SignUpFragment.java]: @at-thachnguyen2 : please edit code in this function for terse

File: app/src/at-thachnguyen/java/asiantech/internship/summer/thachnguyen/debug/menu/MenuActivity.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+import asiantech.internship.summer.thachnguyen.debug.activity_and_fragment.ActivityAndFragment;
+import asiantech.internship.summer.R;
+
+public class MenuActivity extends AppCompatActivity implements View.OnClickListener {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_menu);
+        Button btnEx1 = findViewById(R.id.btnEx1);
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/java/asiantech/internship/summer/thachnguyen/debug/menu/MenuActivity.java]: @at-thachnguyen2 : please change name button for clear mean

File: app/src/at-thachnguyen/res/anim/slide_left_in.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shareInterpolator="false">
+    <translate
+        android:duration="@integer/default_transaction_animation_duration"
+        android:fromXDelta="-100%"
+        android:fromYDelta="0%"
+        android:toXDelta="0%"
+        android:toYDelta="0%" />
+</set>
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/res/anim/slide_left_in.xml]: @at-thachnguyen2 : please fix it

File: app/src/at-thachnguyen/res/anim/slide_left_out.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shareInterpolator="false">
+    <translate
+        android:duration="@integer/default_transaction_animation_duration"
+        android:fromXDelta="0%"
+        android:fromYDelta="0%"
+        android:toXDelta="-100%"
+        android:toYDelta="0%" />
+</set>
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/res/anim/slide_left_out.xml]: @at-thachnguyen2 : please fix it

File: app/src/at-thachnguyen/res/anim/slide_right_in.xml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shareInterpolator="false">
+    <translate
+        android:duration="@integer/default_transaction_animation_duration"
+        android:fromXDelta="100%"
+        android:fromYDelta="0%"
+        android:toXDelta="0%"
+        android:toYDelta="0%" />
+</set>
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/res/anim/slide_right_in.xml]: @at-thachnguyen2 : please fix it

File: app/src/at-thachnguyen/res/anim/slide_right_out.xml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shareInterpolator="false">
+    <translate
+        android:duration="@integer/default_transaction_animation_duration"
+        android:fromXDelta="0%"
+        android:fromYDelta="0%"
+        android:toXDelta="100%"
+        android:toYDelta="0%" />
+</set>
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/res/anim/slide_right_out.xml]: @at-thachnguyen2 : please fix it

File: app/src/at-thachnguyen/res/layout/activity_menu.xml [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:scrollbars="vertical"
+    tools:context=".thachnguyen.debug.menu.MenuActivity">
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+        <Button
+            android:id="@+id/btnEx1"
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/res/layout/activity_menu.xml]: @at-thachnguyen2 : please id name for clear mean

File: app/src/at-thachnguyen/res/layout/fragment_login.xml [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+        android:id="@+id/tvSignUp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="30dp"
+        android:text="@string/sign_up_request"
+        android:textColor="@color/colorWhite"
+        android:textSize="24sp"
+        android:textStyle="bold" />
+    <TextView
+        android:id="@+id/tvLogIn"
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/res/layout/fragment_login.xml]: @at-thachnguyen2 : change tvLogIn -> tvLogin

File: app/src/at-thachnguyen/res/values/strings.xml [hunk 10]
```diff
@@ -1,3 +1,13 @@
 <resources>
     <string name="app_name">Intern18Summer</string>
+    <string name="ex1">View and View Group</string>
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/res/values/strings.xml]: @at-thachnguyen2 : please name for clear mean

**Comment by at-thachnguyen2** [comment]
@at-hanhnguyen2  I fixed your comments. Please, can you check them?  Thank you so much

File: app/src/at-thachnguyen/java/asiantech/internship/summer/activity_and_fragment/LoginFragment.java [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+import android.widget.Toast;
+
+import java.util.Objects;
+
+import asiantech.internship.summer.R;
+
+public class LoginFragment extends Fragment {
+    static final String DATA_RECEIVE_EMAIL = "email";
+    static final String DATA_RECEIVE_PASSWORD = "password";
+    private Toolbar toolbar;
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/java/asiantech/internship/summer/activity_and_fragment/LoginFragment.java]: @at-thachnguyen2 : please set name view follow convention

File: app/src/at-thachnguyen/java/asiantech/internship/summer/activity_and_fragment/SignUpFragment.java [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+import android.view.ViewGroup;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+import java.util.Objects;
+import asiantech.internship.summer.R;
+
+public class SignUpFragment extends Fragment {
+    private static final String DATA_RECEIVE_EMAIL = "mail";
+    private EditText edtEmail;
```
Comment (at-hanhnguyen2) [inline app/src/at-thachnguyen/java/asiantech/internship/summer/activity_and_fragment/SignUpFragment.java]: @at-thachnguyen2 : please set name view follow convention

**Comment by at-thachnguyen2** [comment]
br @at-hanhnguyen2 I fixed your comments. please, can you check them? thank you so much

**Pull request merged**